### ISSUE TITLE
Back button can resume an old version of a task

### ISSUE THREAD

**Issue by cromoteca** [dmfs/tasks#122]
Hi, I have a Sony Xperia V with ROM 4.3. I edited a task, then I saved it. I can't remember the exact sequence of events, anyway I pressed the system back button some times to go back to the task list, but it went to other tasks instead. Also, one of these was the same task I just edited, but with the old information. Pressing back again saved the old data, thus discarding my last changes.

For some reason, there is a stack of calls to the app that gets emptied when the system back button is used. Important: I always enter the app from the widget.

**Comment by dmfs** [comment]
You're right, I can reproduce this mess. I guess we need to set the editor launch mode to "singleTop" or "singleInstance" to make sure only one editor can exist at a time.
The problem is that the editor content is not reloaded automatically when you edit the task in another editor instance. That would be difficult if you already modified some fields. So allowing only one editor instance is probably the best way to deal with that.
